preupg/creator/ui_helper.py

```diff
@@ -133,7 +124,37 @@ def specify_upgrade_path(self):
         return True
 
     def prepare_content_env(self):
-        self.content_path = os.path.join(self.get_group_name(), self.get_content_name())
+        self.content_path = os.path.join(self.get_group_name(),
+                                         self.get_content_name())
+
+    def ask_about_properties_ini(self):
```

Comment:
Now you have function _ask_about_properties_ini_ that does not ask user anything directly yet but the function that asks is named _get_properties_ini_versions_.
What about:
- _get_properties_ini_info_ for this one (or _gather_ instead of _get_). Because in case the _properties.ini_ already exists then the function at least gets the path to the file.
- _ask_about_properties_ini_versions_ for the one that asks user.